src/derived.rs

```diff
@@ -46,6 +54,8 @@ where
     DB: Database,
 {
     fn should_memoize_value(key: &Q::Key) -> bool;
+
+    fn is_volatile(key: &Q::Key) -> bool;
```

Comment:
Let's add a doc-comment explaining what `volatile` means? Or perhaps just rename to `should_track_dependencies`?

Reply:
Sounds good..